### ISSUE TITLE
About training time

### ISSUE THREAD

**Issue by CHANTILLY2023** [dk-liang/UniSeg3D#9]
Thank you for your excellent work. I would like to ask, how long does the training take?

**Comment by xuweigogogo** [comment]
> Thank you for your excellent work. I would like to ask, how long does the training take?

Hi, we conduct experiments on a single RTX 4090 GPU, each training phase taking ~48 hours.

**Comment by CHANTILLY2023** [comment]
Thank you very much for your information!

Thank you so much! Could you please advise if we can download only specific file types (e.g., certain suffixes) instead of the entire 1.2TB ScanNet V2 dataset? This would help us manage storage better.

**Comment by xuweigogogo** [comment]
> Thank you very much for your information!
> 
> Thank you so much! Could you please advise if we can download only specific file types (e.g., certain suffixes) instead of the entire 1.2TB ScanNet V2 dataset? This would help us manage storage better.

Certainly, if your goal is model training without conducting OV testing, you may choose not to download the `'.sens'` part. To achieve this, you can specify the `FILETYPES` parameter in the scannetv2 download code as 
`FILETYPES = [
'.aggregation.json',
'.txt',
'_vh_clean_2.0.010000.segs.json',
'_vh_clean_2.ply',
'_vh_clean_2.labels.ply'
]`.

**Comment by CHANTILLY2023** [comment]
> > Thank you very much for your information!
> > Thank you so much! Could you please advise if we can download only specific file types (e.g., certain suffixes) instead of the entire 1.2TB ScanNet V2 dataset? This would help us manage storage better.
> 
> Certainly, if your goal is model training without conducting OV testing, you may choose not to download the `'.sens'` part. To achieve this, you can specify the `FILETYPES` parameter in the scannetv2 download code as `FILETYPES = [ '.aggregation.json', '.txt', '_vh_clean_2.0.010000.segs.json', '_vh_clean_2.ply', '_vh_clean_2.labels.ply' ]`.

Thank you very much for your advice! I have downloaded the 5 file types you mentioned, but the '.sens' file is too large, with a single file exceeding 1GB, making it impossible for me to download it completely. I was wondering if you could kindly share a '.sens' file with me? My email address is gangzhu1.0@qq.com. Additionally, I’m looking forward to the release of the OV segmentation script. Thank you once again for your help!

**Comment by xyb1314** [comment]
Hi! I would like to ask some code questions! if i would like to inference on single sence (ply/ pointcloud), how do i modify the codes! I have modified the scannetv2_val.txt scannetv2_test.txt. but the outcome seems in vain！

**Comment by xuweigogogo** [comment]
> Hi! I would like to ask some code questions! if i would like to inference on single sence (ply/ pointcloud), how do i modify the codes! I have modified the scannetv2_val.txt scannetv2_test.txt. but the outcome seems in vain！

We write validation scenes in `uniseg3d_infos_val.pkl`. You can modify this file to specify validation scenes.

**Comment by xyb1314** [comment]
> > Hi! I would like to ask some code questions! if i would like to inference on single sence (ply/ pointcloud), how do i modify the codes! I have modified the scannetv2_val.txt scannetv2_test.txt. but the outcome seems in vain！
> 
> We write validation scenes in `uniseg3d_infos_val.pkl`. You can modify this file to specify validation scenes.

thanks! I have maked it ! what is the output for inference ? It seems that only inferencing log might be saved , and how to obtain the mask information for pointcloud seg? like point cloud coordinates or something?
